x-pack/plugins/ml/public/application/data_frame_analytics/pages/job_map/job_map.tsx

```diff
@@ -151,20 +127,63 @@ export const JobMap: FC<Props> = ({ analyticsId, modelId }) => {
     <>
       <EuiSpacer size="m" />
       <div style={{ height: height - parseInt(theme.gutterTypes.gutterLarge, 10) }} ref={ref}>
```

Comment:
On my setup, I have to subtract off another `20px` here to avoid the outer scrollbar for the bottom gutter spacing:

```
height: height - parseInt(theme.gutterTypes.gutterLarge, 10) -20
```